client/go/cmd/command_tester.go

```diff
@@ -76,7 +78,10 @@ func execute(cmd command, t *testing.T, client *mockHttpClient) (string, string)
 
 	// Execute command and return output
 	rootCmd.SetArgs(append(cmd.args, cmd.moreArgs...))
-	Execute()
+	err := Execute()
+	if cmd.failTestOnError {
+		require.Nil(t, err)
```

Comment:
Shouldn't err be non-nil? `assert.NotNil(t, err)`?

Reply:
The intent here is that you ca set failTestOnError to true if you want the test to fail if execute() returns an error.

The best solution here is probably to invert it and create a `failTestOnSuccess` so that tests that should fail on execution was explicitly marked as such, but there were too many cases of that to add that to this PR.